spec/spec_helper.rb

```diff
@@ -45,6 +91,22 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+    start_es_server unless ENV['ES_SKIP']
```

Comment:
ah that is interesting! I did not try it, I always thought that `suite` was before running all specs rather than each one. Very interesting!